libs/langchain/langchain/vectorstores/redis.py

```diff
@@ -191,10 +201,11 @@ def _create_index(self, dim: int = 1536) -> None:
 
         # Check if index exists
         if not _check_index_exists(self.client, self.index_name):
+            metadata_textfields = [field_dict[v](name=k) for k,v in self.metadata_keys.items()]
```

Comment:
I recommend renaming this variable to be `metadata_fields` or `metadata`. The current name suggests the types are all `TextField`